tests/Endpoints/SimilarDocumentsTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Endpoints;
+
+use Meilisearch\Contracts\SimilarDocumentsQuery;
+use Meilisearch\Endpoints\Indexes;
+use Tests\TestCase;
+
+final class SimilarDocumentsTest extends TestCase
+{
+    private Indexes $index;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->index = $this->createEmptyIndex($this->safeIndexName());
+        $this->index->updateDocuments(self::DOCUMENTS);
+        $promise = $this->index->updateFilterableAttributes(['genre']);
+        $this->index->waitForTask($promise['taskUid']);
+    }
+
+    public function testBasicSearchWithFilters(): void
+    {
+        $response = $this->index->search('prince');
+
+        self::assertGreaterThanOrEqual(1, $response->getHitsCount());
+
+        $documentId = $response->getHit(0)['id'];
+        $response = $this->index->searchSimilarDocuments(
+            (new SimilarDocumentsQuery())
+                ->setId($documentId)
+        );
+
+        self::assertNotNull($response);
```

Comment:
Can we check we have some expected fields in the answer (specific to similar documents query) even if we don't check the value?

Reply:
can you share the response format of this api, i could not find it.
i tried running the rc1 locally, however i was getting "Cannot find embedder with name `default`"

Reply:
I will ask my colleagues next week for having a basic tests with basic documents to make it test simply 😊

Reply:
Hello!

To use the similar endpoints, you **need** to:

1. enable the `vectorStore` experimental feature
2. declare at least one embedder in your configuration

For quick sanity tests, the simplest is probably to declare a `userProvided` embedder.

The settings could be:

```jsonc
"embedders": {
            "manual": {
                "source": "userProvided",
                "dimensions": 3,
            }
        }
```

Then declare a set of documents that contain the `_vectors.manual` field, for example:

```jsonc
[
        {
            "title": "Shazam!",
            "release_year": 2019,
            "id": "287947",
            // Three semantic properties:
            // 1. magic, anything that reminds you of magic
            // 2. authority, anything that inspires command
            // 3. horror, anything that inspires fear or dread
            "_vectors": { "manual": [0.8, 0.4, -0.5]},
        },
        {
            "title": "Captain Marvel",
            "release_year": 2019,
            "id": "299537",
            "_vectors": { "manual": [0.6, 0.8, -0.2] },
        },
        {
            "title": "Escape Room",
            "release_year": 2019,
            "id": "522681",
            "_vectors": { "manual": [0.1, 0.6, 0.8] },
        },
        {
            "title": "How to Train Your Dragon: The Hidden World",
            "release_year": 2019,
            "id": "166428",
            "_vectors": { "manual": [0.7, 0.7, -0.4] },
        },
        {
            "title": "All Quiet on the Western Front",
            "release_year": 1930,
            "id": "143",
            "_vectors": { "manual": [-0.5, 0.3, 0.85] },
        }
    ]
```

The example are from the [the engine test file](https://github.com/meilisearch/meilisearch/blob/main/meilisearch/tests/similar/mod.rs#L9) for the similar endpoints.

> can you share the response format of this api, i could not find it.

You're right I added the response format to [the public usage page](https://meilisearch.notion.site/Get-similar-documents-usage-540919ca755c4da0b7cdee273db3f290#f34dc66d893e42a7a6fe320efdc60c65), please feel free to ask for more information as needed :sunny: